release_notes_2.0.md

```diff
@@ -0,0 +1,119 @@
+# Release notes 2.0
+
+TODO:
+
+- [ ] Open Enclave?
+- [ ] TLS?
+- [ ] Commit evidence?
+
+---
+
+## Developer API
+
+### C++
+
+- CCF is now built with Clang 10. It is recommended that C++ applications upgrade to Clang 10 as well.
+- Raised the minimum supported CMake version for building CCF to 3.16 (#2946).
+
+- Added `get_untrusted_host_time_v1` API. This can be used to retrieve a timestamp during endpoint execution, accurate to within a few milliseconds. Note that this timestamp comes directly from the host so is not trusted, and should not be used to make sensitive decisions within a transaction (#2550).
+- Added `get_quotes_for_all_trusted_nodes_v1` API. This returns the ID and quote for all nodes which are currently trusted and participating in the service, for live audit (#2511).
+- Added `get_metrics_v1` API to `BaseEndpointRegistry` for applications that do not make use of builtins and want to version or customise metrics output.
+- `ccf::historical::adapter_v2` now returns 404, with either `TransactionPendingOrUnknown` or `TransactionInvalid`, rather than 400 when a user performs a historical query for a transaction id that is not committed.
+- `ccf::historical::AbstractStateCache::drop_requests()` renamed to `drop_cached_states()` (#3187).
+- `get_state_at()` now returns receipts for signature transactions (#2785), see [documentation](https://microsoft.github.io/CCF/main/use_apps/verify_tx.html#transaction-receipts) for details.
+
```

Comment:
```suggestion
- Added indexing system to speed up historical queries (#3280, #3444).
```

Reply:
A link to the doc page would be really good "See documentation for details".